src/core/pages/AccountInfo.vue

```diff
@@ -156,13 +159,13 @@ export default {
       try {
         this.loading = true;
 
-        this.$v.userProfile.local.password.$touch();
-        this.$v.passwordConfirm.$touch();
-        if (this.$v.userProfile.local.password.$error || this.$v.passwordConfirm.$error) {
+        this.$v.newPassword.password.$touch();
+        this.$v.newPassword.passwordConfirm.$touch();
+        if (this.$v.newPassword.password.$error || this.$v.newPassword.passwordConfirm.$error) {
```

Comment:
simplement `this.$v.newPassword.$error` car si `password` ou `passwordConfirm` sont en erreur l'objet parent le sera egalement

Reply:
done